app/bartender-client/constants.ts

```diff
@@ -1,3 +1,4 @@
 export const BARTENDER_APPLICATION_NAME = "haddock3";
 export const WORKFLOW_CONFIG_FILENAME = "workflow.cfg";
 export const JOB_OUTPUT_DIR = "output";
+export const PAGE_REFRESH_RATE_MS = 5000;
```

Comment:
Nice that the unit is included in the name, I always think its in seconds like with Python or shell.